_docs-sources/refarch/access/how-to-auth-vpn/index.md

```diff
@@ -1,9 +1,44 @@
 # VPN Authentcation 
 
+Most of the AWS resources that comprise the Reference Architecture run in private subnets, which means they do not have a public IP address, and cannot be reached directly from the public Internet. This reduces the "surface area" that attackers can reach. Of course, you still need access into the VPCs, so we exposed a single entrypoint into the network: an [OpenVPN server](https://openvpn.net/).
 
-Haxx0r ipsum foo Trojan horse new all your base are belong to us ip error private shell fopen semaphore epoch char packet sniffer segfault gurfle bypass. Memory leak bubble sort injection leet malloc brute force double xss mega sudo mountain dew void echo win emacs linux piggyback bin. I'm compiling float bang case cat infinite loop Donald Knuth unix for /dev/null machine code then chown d00dz worm gnu crack packet bar eof while.
+You use an OpenVPN client to connect to the OpenVPN server, which gets you "in" to the network, allowing you to securely connect to private resources in the account.
 
-Lib void brute force bypass nak concurrently all your base are belong to us break leapfrog bit default packet sniffer Linus Torvalds. Man pages packet stack trace Starcraft Donald Knuth pwned worm hello world public giga frack gurfle. Irc fork malloc fopen script kiddies flood blob fail hexadecimal while access semaphore loop mega Trojan horse foo gobble.
+Here are the steps you'll need to take:
```

Comment:
Can we re-write this list as a sentence/paragraph, since the sections speak for themselves and directly below?

Reply:
Resolved in 62fe8df61a4e5f904e27d312227fa5acb35af668